add documentation for skip_existing feature for text embedding processor (#9483)
opensearch-project/documentation-website: _ingest-pipelines/processors/text-embedding.md

`if` | String containing a Boolean expression | Optional | A condition for running the processor.|
`ignore_failure` | Boolean | Optional | Specifies whether the processor continues execution even if it encounters an error. If set to `true`, the processor failure is ignored. Default is `false`.|
`on_failure` | List | Optional | A list of processors to run if the processor fails. |

> `skip_existing` | Boolean | Optional | Specifies whether the processor skips making inference call for fields that already have embeddings. If set to `true`, the processor will not make inference call on fields where the embeddings haven't changed. Default is `false`.|

Reviewer: ```suggestion
`skip_existing` | Boolean | Optional | When `true`, the processor does not make inference calls for fields that already contain embeddings, leaving existing embeddings unchanged. Default is `false`.|
```
